test/box-luatest/upgrade/2.11.5/replicaset/instance-001/gen.lua

```diff
@@ -0,0 +1,16 @@
+-- Tarantool 2.11.5-entrypoint-103-ga2cc2b0080. Before release of the 2.11.5.
+box.cfg{
+    replicaset_uuid = 'b5c6e102-aa65-4b5f-a967-ee2a4f5d1480',
+    instance_uuid = '22188ab6-e0e5-484c-89b6-5dfe2b969b7b',
+    replication = {3301, 3302},
+    listen = 3301,
+}
+
+box.schema.user.grant('guest', 'super')
+while not box.info.replication[2] or
+      not box.info.replication[2].downstream or
+      box.info.replication[2].downstream.status ~= 'follow' do
+    require('fiber').yield(0.1)
```

Comment:
A little typo: `fiber.yield()` doesn't accept an argument: `fiber.sleep()` does,

https://github.com/tarantool/tarantool/blob/9a17476508ce0eb2562dfff1357281e25e274e15/src/lua/fiber.c#L728-L734

Reply:
That's really minor. And this is the only comment. I'll close it and proceed.